
<!doctype html>
<html lang="en">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
<script type="text/javascript" src="../25815a6dd8d07c10e8a6399806c4f922.js"></script>  
<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-K4NS9FB');</script>
<!-- End Google Tag Manager -->  
		<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="" />
  <link rel="shortcut icon" href="../media/universityofexeter/webteam/styleassets/images/favicon.ico" />
<title> Planning your experiment  | Sequencing Facility | University of Exeter</title>
	<link rel="canonical" href="http://biosciences.exeter.ac.uk/sequencing/experiment.html" />
<!--<link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css" rel="stylesheet">-->
  <link href="../www-sub/codebox/bootstrap337/css/bootstrap.min.css" rel="stylesheet">
<link href="../font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet">
<!--<link href='//fonts.googleapis.com/css?family=Droid+Sans:400,700' rel='stylesheet' type='text/css'>-->
  <link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Outfit:wght@100;300;400;700&display=swap" rel="stylesheet">
  <!-- text/stylesheet from Universal config --><link href="../media/universityofexeter/css/generic-branding.css" rel="stylesheet">
  <!-- text/addstylesheet from Universal config --><link href="../media/universityofexeter/css/responsive-branding.css" rel="stylesheet">
<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
 <!-- navigation object : Header code -->
   <!-- Get Global GA code for Universal page layout --> <script>
	var _gaq = _gaq || [];
	_gaq.push(['_setAccount', 'UA-7878092-1']);
	_gaq.push(['_setDomainName', 'exeter.ac.uk']);
	_gaq.push(['_trackPageview']);
	(function() {
	var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
	ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
	var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	})();
	// detect if user prints page
	try{
	(function() {
	var afterPrint = function() {
	_gaq.push(['_trackEvent', 'Print Intent', document.location.pathname]); //for classic GA
	};
	if (window.matchMedia) {
	var mediaQueryList = window.matchMedia('print');
	mediaQueryList.addListener(function(mql) {
	if (!mql.matches)
	afterPrint();
	});
	}
	window.onafterprint = afterPrint;
	}());
	} catch(e) {}
</script>
  <!-- get text/secondga from universal config  gtag('config', ''); -->
    <!-- get text/inhead from universal config --> 
  <!-- Hotjar Tracking Code for undergraduate traffic -->
<script>
    (function(h,o,t,j,a,r){
        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
        h._hjSettings={hjid:981027,hjsv:6};
        a=o.getElementsByTagName('head')[0];
        r=o.createElement('script');r.async=1;
        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
        a.appendChild(r);
    })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
</script>
<link rel="stylesheet" type="text/css" href="../www-sub/codebox/cookie-policy-notification/styles.css" />
<script src="../ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
<script src="../www-sub/codebox/cookie-policy-notification/notification.js"></script>
   <style>
	 a#skip-to-content{
    left:-999px;
    position:absolute;
    top:auto;
    width:1px;
    height:1px;
    overflow:hidden;
    z-index:-999;
}
a#skip-to-content:focus, a#skip-to-content:active {
    color: #fff;
    background-color:#000;
    left: auto;
    top: auto;
    width: 30%;
    height: auto;
    overflow:auto;
    margin: 10px 35%;
    padding:5px;
    border-radius: 15px;
    border:4px solid yellow;
    text-align:center;
    font-size:1.2em;
    z-index:999;
}
a:focus {
    outline: thin dotted;
    outline-offset: 0px;
}	 
</style>
</head>
<body>
<!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-K4NS9FB" height="0" width="0" style="display:none;visibility:hidden" aria-hidden="true" title="Google tag manager"></iframe></noscript>
<!-- End Google Tag Manager (noscript) --> 
  <a id="skip-to-content" class="sr-only sr-only-focusable" href="#left-col">Skip to main content</a>
    <!-- get text/startbody from universal config --> 
  <header id="header-area">
  <!-- Starts -->
  <div class="container"> 
 <!-- get Get Global header for Universal page layout-->  <div class="header-top">
      <!-- Header Links Starts -->
        <div class="header-links">
          <div class="navbar-header">
            <button type="button" class="btn btn-navbar navbar-toggle" data-toggle="collapse" data-target=".header-links-collapse" tabindex="0" aria-label="global links"> <span class="sr-only">Global links </span> <i class="fa fa-bars"></i> </button>
          </div>
          <div class="collapse navbar-collapse header-links-collapse">
            <ul class="nav navbar-nav">
              <li> <a href="//www.exeter.ac.uk">University home</a> </li>
              <li> <a href="https://www.exeter.ac.uk/study/">Study</a> </li>
              <li> <a href="https://www.exeter.ac.uk/research/">Research</a> </li>
              <li> <a href="https://www.exeter.ac.uk/business/">Business</a> </li>
              <li> <a href="https://www.exeter.ac.uk/alumnisupporters/">Alumni and supporters</a> </li>
              <li> <a href="https://www.exeter.ac.uk/departments/">Our departments</a> </li>
              <li> <a href="https://www.exeter.ac.uk/visit/">Visiting us</a> </li>
              <li> <a href="https://www.exeter.ac.uk/about/">About us</a> </li>
            </ul>
          </div>
      </div>
      <!-- Header Links Ends --> 
  </div>
  <!-- Header Top Ends --> 
    <!-- Main Header Starts -->
    <div class="main-header">
      <div class="row"> 
        <!-- Logo Starts -->
       <!-- pull in /sitetitle -->
        <div class="col-md-12"><div class="pull-left"> <a href="../index.html"><img src="../www-sub/media/universityofexeter/webteam/styleassets/images/UOE-green.svg" alt="University of Exeter" id="logo" />
          <span id='logo-text'>Sequencing Facility</span></a></div>
          <div class="pull-right" id="right-contents">
            <div id="sm_icons">
               <p><!-- navigation object : Config PL: Get intranet links --></p>
                    <!-- pull in text/pl_social --><p><a href="http://ess-portal.exeter.ac.uk/signin" class="intranet-links">Log in to the Sequencing Facility portal</a><!-- 1 -->
             </div>
        <!-- pull in text/pl_search 2 --> <form class="navbar-form" role="search" action="https://search.exeter.ac.uk/s/search.html" method="get"><input type="hidden" value="all-exeter-uni" name="collection"><input type="hidden" name="form" value="simple"><div class="input-group"><input type="text" class="form-control" placeholder="Search website" name="query" id="query" aria-label="Search website"><div class="input-group-btn"><button class="btn btn-default" type="submit" aria-label="search website"><i class="glyphicon glyphicon-search"></i></button></div> </div></form>
          </div>
        </div>
        <!-- Logo Ends --> 
      </div>
    </div>
    <!-- Main Header Ends --> 
    <!-- Main Menu Starts -->
    <nav id="main-menu" class="navbar" role="navigation"> 
      <!-- Nav Header Starts -->
      <div class="navbar-header">
        <button type="button" class="btn btn-navbar navbar-toggle" data-toggle="collapse" data-target=".navbar-cat-collapse" aria-label="website links"> <span class="sr-only">Website links </span> <i class="fa fa-bars"></i> </button>
      </div>
      <!-- Nav Header Ends --> 
      <!-- Navbar Cat collapse Starts -->
      <div class="collapse navbar-collapse navbar-cat-collapse">
          <!-- navigation object : get menu from universal configurator--><ul class='nav navbar-nav'><li><a href="../index.html">Home</a></li><li><a href="../sequencing/equipment.html">Equipment and resources</a>
<ul class="multilevel-linkul-0">
<li><a href="../sequencing/equipment/illumina.html">Illumina Sequencing</a></li>
<li><a href="../sequencing/equipment/pacbio.html">Long Read Sequencing</a></li>
<li><a href="../sequencing/equipment/nanopore.html">Oxford Nanopore Sequencing</a></li>
</ul>
</li><li><span class="currentbranch0"><a href="../sequencing/experiment.html">Your experiment</a></span>
<ul class="multilevel-linkul-0">
<li><span class="currentbranch1"><a href="../sequencing/experiment/plan.html">Planning your experiment</a></span></li>
<li><a href="../sequencing/experiment/samples.html">Preparing your samples for sequencing</a></li>
<li><a href="../sequencing/experiment/submitsamples.html">How to submit your samples</a></li>
<li><a href="../sequencing/experiment/bioinformatics.html">Bioinformatics support</a></li>
</ul>
</li><li><a href="../sequencing/pricing.html">Pricing</a>
<ul class="multilevel-linkul-0">
<li><a href="../sequencing/pricing/options.html">Options for working with us</a></li>
<li><a href="../sequencing/pricing/grants.html">Grant application costings</a></li>
<li><a href="../sequencing/pricing/internal.html">Internal costs</a></li>
<li><a href="../sequencing/pricing/commercial.html">Commercial costs</a></li>
</ul>
</li><li><a href="../sequencing/termsandfaqs.html">Terms and FAQs</a>
<ul class="multilevel-linkul-0">
<li><a href="../sequencing/termsandfaqs/terms.html">Terms of service</a></li>
<li><a href="../sequencing/termsandfaqs/faqs.html">Frequently asked questions</a></li>
<li><a href="../sequencing/termsandfaqs/acknowledging.html">Acknowledging the service</a></li>
</ul>
</li><li><a href="../sequencing/contact.html">Contact</a></li></ul><!-- level7 -->
      </div>
      <!-- Navbar Cat collapse Ends --> 
    </nav>
    <!-- Main Menu Ends --> 
  </div>
  <!-- Ends --> 
</header>
      <div class="container" id="main-content">
  <div class="row"> 
    <!-- Sidebar Starts -->
    <div class="col-sm-3 col-sm-push-9" id="right-col">
      <div class="panel panel-default" id="sub-nav">
        <div class="panel-body">
          <div class="navbar-header">
            <button type="button" class="btn btn-navbar navbar-toggle" data-toggle="collapse" data-target=".sub-nav-links" aria-label="section links"> <span class="sr-only">Section links </span> <i class="fa fa-bars"></i> </button>
          </div>
          <div class="collapse navbar-collapse sub-nav-links">
            		<!-- navigation object : universal config  Sub nav --><ul class='menu'></ul>          
            </div>
      </div>
        </div>
        <!-- navigation object : Right column -->
    </div>
    <!-- Sidebar Ends --> 
    <!-- Primary Content Starts -->
    <div class="col-sm-9 col-sm-pull-3" id="left-col">
      <!-- navigation object : Breadcrumb nav from universal config text/pl_breadcrumb --><div class='breadcrumb'><a href='../index.html'>Home</a><a href="../sequencing/experiment.html">Your experiment</a><a href="../sequencing/experiment/plan.html">Planning your experiment</a></div>
<h1>Planning your experiment</h1>
 <!--
 <h1 class="sr-only">Planning your experiment</h1>
 -->
 <h1 class="sr-only">Planning your experiment</h1>
 <h1 class="sr-only">Planning your experiment</h1>
            <!-- navigation object : Left column --><p>Careful experimental planning is crucial to obtain data which can yield the information you need for your project. For most experiments the most pressing question relates to the amount and type of sequencing required per sample, but there are other factors which may impact your experimental design.</p>
<p>If in any doubt, please do <a href="../sequencing/contact.html">discuss your project with us</a>. Protocols change frequently and it is possible that improved methods are available which may help your project.</p>
<h2>Genomic sequencing</h2><!-- Accordion starts --> 
<div class="panel-group accordion" id="accordion0"><div class="panel panel-default">
    <div class="panel-heading" data-toggle="collapse" data-target="#collapse0-0" data-parent="#accordion0">
      <p class="panel-title">
        <a class="accordion-toggle" href="#0">
          How much sequencing do I need?
        </a><span class="indicator glyphicon glyphicon-chevron-down  pull-right" aria-hidden="true"></span>
      </p>
    </div>
    <div id="0-0" class="panel-collapse collapse">
      <div class="panel-body">
<p>This depends on the size of your genome, its ploidy and whether you intend to compare against a reference genome or perform de-novo assembly. It can also be affected by biases in sample preparation (e.g. GC-bias) so the figures below are only a general guide.</p>
<h3>Reference-based:</h3>
<p>For haploid genomes with a single copy of major chromosomes (e.g. bacteria), aim for a median coverage of 10x (mean coverage of 15-20x).</p>
<p>For diploids, this should be doubled: i.e. you should aim for median coverage of your genome of 20-30x. In practice this means a mean depth of coverage of 40-50x. This has been shown to enable relatively accurate variant calling. The following Illumina technical note can be used to calculate the amount of sequence data required for an experiment.</p>
<p>These figures apply whether performing <a href="../sequencing/equipment/illumina.html">Illumina short read sequencing</a> or <a href="../sequencing/equipment/pacbio.html">Long read sequencing</a>.</p>
<h3>De-novo sequencing:</h3>
<p>If you do not have a reference genome (denovo sequencing) then, aim for 100x-200x coverage per sample with&nbsp;<a href="../sequencing/equipment/illumina.html">Illumina sequencing</a>.</p>
<p>If using long read data&nbsp;data then the question is whether an assembly can be achieved with Nanopore only data (preferred) or if the Nanopore data will be used to help scaffold high quality Illumina contigs. In the Nanopore only case, a mean coverage of 50x is recommended when using modern Nanopore aware assemblers (e.g. EPI2ME). In the case of scaffolding, a coverage of 4x should be sufficient to scaffold the majority of the genome.</p>
</div>
    </div>
  </div><div class="panel panel-default">
    <div class="panel-heading" data-toggle="collapse" data-target="#collapse0-1" data-parent="#accordion0">
      <p class="panel-title">
        <a class="accordion-toggle" href="#1">
          What sort of libraries do I need?
        </a><span class="indicator glyphicon glyphicon-chevron-down  pull-right" aria-hidden="true"></span>
      </p>
    </div>
    <div id="0-1" class="panel-collapse collapse">
      <div class="panel-body">
<p>There are many ways to prepare material for sequencing. These range from PCR-free protocols which require large inputs of starting material (&gt;1-2ug) to protocols, such as NEBNext Ultra II FS or QIAgen ultra low input library preparation, which can work with low inputs (10-100ng).</p>
<p>If you are working with an organism with a biased GC-content (i.e. &lt;40% or &gt;60%), we would recommend a PCR-free protocol if possible as the PCR-reaction can lead to over/under representation of certain features. PCR-free libraries require larger input of DNA. Please see the <a href="../sequencing/experiment/samples.html">sample preparation pages</a> for more details of individual protocols.</p>
<h3>De-novo sequencing:</h3>
<p>In addition if you are planning a de-novo sequencing experiment then it is recommended that several libraries are prepared with different insert sizes. We would recommend one library with an insert size of 200-500bp along with a long reads sequencing library (10-20kb fragments ) sequenced by&nbsp;Oxford Nanopore Technologies platform.</p>
</div>
    </div>
  </div></div>
<!-- Accordion ends --><h2>RNA-seq</h2><!-- Accordion starts --> 
<div class="panel-group accordion" id="accordion1"><div class="panel panel-default">
    <div class="panel-heading" data-toggle="collapse" data-target="#collapse1-2" data-parent="#accordion1">
      <p class="panel-title">
        <a class="accordion-toggle" href="#2">
          How much sequencing do I need?
        </a><span class="indicator glyphicon glyphicon-chevron-down  pull-right" aria-hidden="true"></span>
      </p>
    </div>
    <div id="1-2" class="panel-collapse collapse">
      <div class="panel-body">
<p>This depends on the biological variability between replicates as well as the organism. Typically for prokaryote RNA-seq we would suggest aiming for 3-5 million reads per biological replicate and for eukaryotes anywhere from 10-40 million per replicate. Aim for high numbers if you wish to quantify splice variants and/or transcription start sites or if your genes of interest are likely to be expressed in low numbers; for detection of rare varients in human RNAseq 100 million reads/sample may be required.</p>
<p>The&nbsp;<a href="../media/universityofexeter/schoolofbiosciences/responsive/facilities/Power_Calculator.xls">RNA Seq Power calculator spreadsheet</a> may assist you in planning your experiment, but it is only a guide.</p>
<p>We generally recommend a minimum of 4-6 biological replicates per condition assuming a well controlled environment. This allows for the possibility that some replicates may be more variable than you anticipate. If you are dealing with environmental samples where biological variability is much higher you may need double this.</p>
<p>Note that the ONT protocols for cDNA library preparation can only be used to identify full length transcripts. Quantitative information is not provided when the libraries are amplified.</p>
</div>
    </div>
  </div><div class="panel panel-default">
    <div class="panel-heading" data-toggle="collapse" data-target="#collapse1-3" data-parent="#accordion1">
      <p class="panel-title">
        <a class="accordion-toggle" href="#3">
          What sort of libraries do I need?
        </a><span class="indicator glyphicon glyphicon-chevron-down  pull-right" aria-hidden="true"></span>
      </p>
    </div>
    <div id="1-3" class="panel-collapse collapse">
      <div class="panel-body">
<p>The first decision is whether to polyA isolate RNA or to perform an rRNA depletion. This is necessary to remove highly abundant rRNA from your sample.</p>
<p>PolyA isolation has the advantage of being cheaper and potentially introducing less bias at the gene level. However, the polyA isolation method can lead to enrichment of the RNA at the 3&prime; end and can bias isoform estimation and lose any non-poly-adenelated RNA (e.g. histone mRNA).</p>
<p>rRNA depletion is typically more expensive and requires that a suitable method is used to remove rRNA fragments.</p>
<p>For all RNA-seq library preps we strongly recommend the use of a directional library such as&nbsp;<span>Illumina Total RNA Preparation which provides exceptional performance for the analysis of coding and multiple forms of noncoding RNA.</span>&nbsp;Please see the <a href="../sequencing/experiment/samples.html">sample preparation pages</a> for more details of individual protocols.</p>
<p>Note that if you are looking at RNA species smaller than 50-100bp, the standard RNA library preparation methods are not appropriate. These methods tend to lose these smaller fragments during purification processes. Instead a specialised small RNA prep needs to be performed.</p>
</div>
    </div>
  </div></div>
<!-- Accordion ends --><h2>Amplicon sequencing</h2>
<p>This can take many forms (e.g. 16S/18S/Co1), but typically we recommend that you <a href="../sequencing/contact.html">contact us in advance</a>&nbsp;of your experiment to discuss the best methodology. Typically we will suggest that sequencing adaptors and multiplex barcodes are added by the user in a two step PCR.</p>
<p>Note that cluster densities achieved for amplicon sequencing on <a href="../sequencing/equipment/illumina.html">Illumina platforms</a> are typically 30% -60% less than than more complex libraries.</p>
<h2>ChIP-Seq, ATACseq Meth-Seq and RAD-Seq</h2>
<p>For these methods, please <a href="../sequencing/contact.html">contact us directly</a> to discuss your experiment before starting any extraction.</p>
      </div>
      <!-- Primary Content Ends -->   </div>
<!-- #sub-content -->
 <div class="row">
  <!-- navigation object : Sub content -->
</div>
<!-- /#sub-content -->
<!-- navigation object : Get Global footer for Universal page layout--><footer id="footer-area"> 
  <div class="footer-links"> 
    	<div class="pw-widget pw-counter-horizontal">			
            <a class="pw-button-facebook pw-look-native"></a>			
            <a class="pw-button-twitter pw-look-native"></a>			
            <a class="pw-button-linkedin pw-look-native"></a>			
            <a class="pw-button-post-share"></a>		
        </div>
        <div class="pw-widget pw-size-medium pw-counter-show">			
            <a class="pw-button-facebook"></a>			
            <a class="pw-button-twitter"></a>			
            <a class="pw-button-email"></a>			
            <a class="pw-button-post"></a>		
        </div>
      <p><a href="//www.exeter.ac.uk/aboutoursite/">Using our site</a> | <a href="//www.exeter.ac.uk/about/oursite/accessibility/">Accessibility</a> | <a href="//www.exeter.ac.uk/foi/">Freedom of Information</a> | <a href="//www.exeter.ac.uk/dataprotection/">Data Protection</a> | <a href="//www.exeter.ac.uk/copyright/">Copyright & disclaimer</a> | <a href="//www.exeter.ac.uk/privacy/">Privacy & cookies</a></p>
  </div>
  <div class="copyright"> 
      <p>Copyright <a href="//www.exeter.ac.uk">University of Exeter</a>. All rights reserved.</p>
  </div>
</footer>
</div>
<!-- /#main-content -->
<script src="../ajax/libs/jquery/1.11.1/jquery.min.js"></script> 
<script src="../www-sub/codebox/bootstrap337/js/bootstrap.min.js"></script>
<script>  
$(document).ready(function() { 
    // highlight current section in sub nav
	$('ul.multilevel-linkul-0').prev('a').addClass('subnav_highlight')
  // move right panels under main contents on handhelds
    checkSize();
    $(window).resize(checkSize);
	function checkSize(){
		if ($(".collapse").css("display") == "none" ){
			$("#right-col .highlight-button").appendTo("#left-col");
			$("#right-col .panel").appendTo("#left-col");
			$("#left-col #sub-nav").appendTo("#right-col");
		}else{
			$("#left-col .highlight-button").appendTo("#right-col");
			$("#left-col .right-panel").appendTo("#right-col");
		}
	}
  // fix T4 dropdown menus
   $("#main-menu ul li:has(ul)").addClass("dropdown");
   $(".currentbranch0").contents().unwrap(); 
   $("#main-menu li.dropdown>a:first-child").addClass("dropdown-toggle");
  $("#main-menu li.dropdown>a:first-child").attr("data-hover", "dropdown");
  $("#main-menu li.dropdown>a:first-child").attr("data-toggle", "dropdown");
   $("#main-menu li.dropdown ul").addClass("dropdown-menu");
  // # hrefs in top tier drop downs so phone users can access drop downs
  $("li.dropdown a.dropdown-toggle").attr("href", "#")
  $("#main-menu").css("visibility", "visible");
});
</script>
<script src="../ajax/libs/bootstrap-hover-dropdown/2.0.10/bootstrap-hover-dropdown.min.js"></script>
  <!-- navigation object : Footer code -->
<script>
$(document).ready(function() { 
// Accordion
  function toggleChevron(e) {
    $(e.target)
        .prev('#accordion0 .panel-heading')
        .find("i.indicator")
         .toggleClass('glyphicon-chevron-up glyphicon-chevron-down');
	}
	$('#accordion0').on('hidden.bs.collapse', toggleChevron);
	$('#accordion0').on('shown.bs.collapse', toggleChevron);
    // open specific accordion entry if directly linked to
var url = document.location.toString();
if ( url.match('#') ) {
    $('#'+url.split('#')[1]).addClass('in');
}
//remove the need for the accordion pane IDs
$('.accordion').each(function(accordionIndex, ca) {
   // console.log("Accordion: " + accordionIndex);
    $(ca).attr("id", "ac" + accordionIndex);
                   $(ca).find('.panel').each(function(index, element) {
                  //    console.log("Panel: " + index);
                      $(element).find(".panel-heading").attr({
                        'data-target': "#ac"+accordionIndex+"a" + index,
                        'data-parent': "#ac" + accordionIndex
                      });
                      $(element).find(".accordion-toggle").attr("href", "#a" + index);
                      $(element).find(".panel-collapse").attr("id", "ac"+accordionIndex+"a" + index);
                    });
  });
});
</script>
<script>
$(document).ready(function() { 
// Accordion
  function toggleChevron(e) {
    $(e.target)
        .prev('#accordion1 .panel-heading')
        .find("i.indicator")
         .toggleClass('glyphicon-chevron-up glyphicon-chevron-down');
	}
	$('#accordion1').on('hidden.bs.collapse', toggleChevron);
	$('#accordion1').on('shown.bs.collapse', toggleChevron);
    // open specific accordion entry if directly linked to
var url = document.location.toString();
if ( url.match('#') ) {
    $('#'+url.split('#')[1]).addClass('in');
}
//remove the need for the accordion pane IDs
$('.accordion').each(function(accordionIndex, ca) {
   // console.log("Accordion: " + accordionIndex);
    $(ca).attr("id", "ac" + accordionIndex);
                   $(ca).find('.panel').each(function(index, element) {
                  //    console.log("Panel: " + index);
                      $(element).find(".panel-heading").attr({
                        'data-target': "#ac"+accordionIndex+"a" + index,
                        'data-parent': "#ac" + accordionIndex
                      });
                      $(element).find(".accordion-toggle").attr("href", "#a" + index);
                      $(element).find(".panel-collapse").attr("id", "ac"+accordionIndex+"a" + index);
                    });
  });
});
</script>
<!-- navigation object : Siteimprove deep linking nav object --><!-- 218718 -->
 <!--<script src="https://s.po.st/share/script/post-widget.js#publisherKey=idm3d21ublhnvd96afid" type="text/javascript"></script>-->
    <!-- get text/endbody from universal config --> 
<!-- Siteimprove analytics code August 2018 -->
<script type="text/javascript">  
/*<![CDATA[*/
(function() {
var sz = document.createElement('script'); sz.type = 'text/javascript'; sz.async = true;
sz.src = '../js/siteanalyze_51623343.js';
var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(sz, s);
})();
/*]]>*/
</script>
<script>
$(document).ready(function() { 
    // open specific accordion entry if directly linked to
var url = document.location.toString();
if ( url.match('#') ) {
    $('#'+url.split('#')[1]).addClass('in');
}
});
</script>
</body>
</html>